src/database/database-sqlite3.cpp

```diff
@@ -606,3 +606,140 @@ void PlayerDatabaseSQLite3::listPlayers(std::vector<std::string> &res)
 
 	sqlite3_reset(m_stmt_player_list);
 }
+
+/*
+ * Auth database
+ */
+
+AuthDatabaseSQLite3::AuthDatabaseSQLite3(const std::string &savedir) :
+		Database_SQLite3(savedir, "auth"), AuthDatabase()
+{
+}
+
+AuthDatabaseSQLite3::~AuthDatabaseSQLite3()
+{
+	FINALIZE_STATEMENT(m_stmt_read)
+	FINALIZE_STATEMENT(m_stmt_write)
+	FINALIZE_STATEMENT(m_stmt_create)
+	FINALIZE_STATEMENT(m_stmt_delete)
+	FINALIZE_STATEMENT(m_stmt_list_names)
+}
+
+void AuthDatabaseSQLite3::createDatabase()
+{
+	assert(m_database); // Pre-condition
+
+	SQLOK(sqlite3_exec(m_database,
+			      "CREATE TABLE IF NOT EXISTS `auth` ("
+			      "`id` INTEGER PRIMARY KEY AUTOINCREMENT,"
+			      "`name` VARCHAR(32),"
+			      "`password` VARCHAR(512),"
+			      "`privileges` VARCHAR(512),"
+			      "last_login INTEGER);",
+			      NULL, NULL, NULL),
```

Comment:
Wrong formatting. Ignore Lint if they requested this.

Reply:
I guess you'll have to edit it manually until clang adds more options so we can correct the issues with the code formatting rules.

Reply:
if needed add // clang-format off before the function and // clang-format on after